src/usePlanshipCustomer.ts

```diff
@@ -0,0 +1,26 @@
+import { CustomerSubscriptionWithPlan, Entitlements, PlanshipCustomerApi } from '@planship/fetch'
+import { EntitlementsBase } from './types.js'
```

Comment:
Import types? I know we talked about whether this is required but I guess we should be consistent.